homeassistant/components/auth/__init__.py

```diff
@@ -227,6 +231,11 @@ def __init__(self, flow_mgr, store_credentials):
             return self.json_message('User input malformed', 400)
```

Comment:
Is this an invalid login? This might be if a user does not fill in a username or password. Or if an MFA module requires numbers and we put in a string.

Reply:
Required field and simple format check for example numbers check, length check etc should done in frontend, if it is still malformed, someone is trying to hack in.